mod/01_rating/set/type/metric.rb

```diff
@@ -16,6 +17,64 @@
   def view_caching?
     true
   end
+
+  view :value_type_edit_modal_link do |args|
```

Comment:
if we're only changing args, this can be in an args method rather than a view.  (eg default_modal_link_args)